sql/search/search_rfm/init.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE TABLE `moz-fx-data-shared-prod.search.search_rfm`
```

Comment:
`init.sql` is only needed when query.sql depends on past results, so this file should be removed